_posts/05-04-01-Design-Patterns.md

```diff
@@ -1,18 +1,18 @@
 ---
 isChild: true
-title: Dizajn paterni (šabloni)
+title: Dizajn šabloni (Design patterns)
 anchor: design_patterns
 ---
 
-## Dizajn paterni (šabloni) {#design_patterns_title}
+## Dizajn šabloni (Design patterns) {#design_patterns_title}
```

Comment:
Iskreno nisam siguran, kad malo bolje razmislim mozda je ovo gore najbolje resenje da imamo obe opcije. Ja sam za to da ga ostavimo onda ovako ;)